Sources/MockoloFramework/Utils/SwiftSyntaxExtensions.swift

```diff
@@ -8,7 +8,40 @@
 import Foundation
 import SwiftSyntax
 
-class EntityVisitor: SyntaxVisitor {
+extension SyntaxParser {
+    public static func parse(_ fileData: Data, path: String,
+                             diagnosticEngine: DiagnosticEngine? = nil) throws -> SourceFileSyntax {
+        // Avoid using `String(contentsOf:)` because it creates a wrapped NSString.
+        let source = fileData.withUnsafeBytes { buf in
+            return String(decoding: buf.bindMemory(to: UInt8.self), as: UTF8.self)
```

Comment:
`init?(data: Data, encoding: String.Encoding)` not simpler? docs are empty though, so hard to say if this ends up getting copied or not.

Reply:
In this case Data will be copied and then a new String instance is created with it. Also this is from one of swiftsyntax parse functions minus some overhead so would like to keep it as close to the original func as possible.